pkgs/development/libraries/vtk/generic.nix

```
@@ -89,6 +89,13 @@ in stdenv.mkDerivation rec {
     sed -i 's/fprintf(output, shift)/fprintf(output, "%s", shift)/g' ./ThirdParty/libxml2/vtklibxml2/xpath.c
   '';
 
+  postInstall = optionalString enablePython ''
+    substitute \
+      ${./vtk.egg-info} \
+      $out/lib/python${pythonInterpreter.pythonVersion}/site-packages/vtk-${version}.egg-info \
+      --subst-var-by VTK_VER "${version}"
```

```suggestion
    substitute ${./vtk.egg-info} \
      $out/${pythonInterpreter.sitePackages}/vtk-${version}.egg-info \
      --subst-var-by VTK_VER "${version}"
```


also pythonInterpreter should be renamed to python